analyzer/runtime/oasis_encryption.go

```diff
@@ -0,0 +1,65 @@
+package runtime
+
+import (
+	"fmt"
+
+	"github.com/oasisprotocol/oasis-core/go/common/cbor"
+	sdkTypes "github.com/oasisprotocol/oasis-sdk/client-sdk/go/types"
+
+	"github.com/oasisprotocol/nexus/common"
+)
+
+type OasisEncryptedData struct {
+	Format      common.CallFormat
+	PublicKey   []byte
+	DataNonce   []byte
+	DataData    []byte
+	ResultNonce []byte
+	ResultData  []byte
+}
```

Comment:
The evmEncrypted and oasisEncrypted tx data share almost all of the same fields; would it make more sense to factor this into something like `common.EncryptedTxData` and use it in both cases?

Reply:
oh yeah we can make a struct just for envelope stuff. but I'm in favor of storing these separately. gonna leave a comment on why